<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1620</width>
    <height>875</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>875</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1620</width>
    <height>875</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Agent</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>871</width>
     <height>821</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 249, 249);
color: rgb(21, 21, 21);
border-radius:10px;</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="plainText">
    <string>$[Agent]</string>
   </property>
   <property name="tabStopWidth">
    <number>30</number>
   </property>
   <property name="placeholderText">
    <string>Insert code here...</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>661</width>
     <height>871</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(75, 75, 75);
color: rgb(181, 181, 181);
border-radius:15px;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="tabStopWidth">
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-expand-&gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>840</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(143, 143, 143);</string>
   </property>
   <property name="text">
    <string>   &lt; Line &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(143, 143, 143);</string>
   </property>
   <property name="text">
    <string>SEARCH:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
